python/ql/test/experimental/dataflow/coverage/test.py

```diff
@@ -203,12 +203,12 @@ def test_nested_comprehension_paren():
 # Iterable unpacking in comprehensions
 def test_unpacking_comprehension():
     x = [a for (a, b) in [(SOURCE, NONSOURCE)]]
-    SINK(x[0])  # Flow missing
+    SINK(x[0]) #$ MISSING:flow="SOURCE, l:-1 -> x[0]"
```

Comment:
I'm surprised we're not able to handle this one now. can you explain why? :confused:

Reply:
Thanks! I had missed this bit..